skimage/measure/fit.py

```diff
@@ -868,7 +868,7 @@ def ransac(data, model_class, min_samples, residual_threshold,
                 break
 
     # estimate final model using all inliers
-    if best_inliers is not None:
+    if best_inliers is not None and sum(best_inliers) > 0:
```

Comment:
It will be more efficient to use `any` rather than `sum` (any will stop checking values as soon as the first True value is found)